nornir_utils/plugins/functions/print_stat.py

```diff
@@ -0,0 +1,96 @@
+import threading
+from typing import Tuple
+from nornir_utils.plugins.functions.print_result import _get_color
+
+from colorama import Fore, Style, init
+
+from nornir.core.task import AggregatedResult, MultiResult, Result
+
+
+LOCK = threading.Lock()
+
+
+init(autoreset=True, strip=False)
```

Comment:
this is why the tests are failing, we are initializing colorama twice. Remove this line and the tests will all pass (you will need to fix your tests)

Reply:
maybe move the other init call to __init__ to have it in a central place

Reply:
moved `colorama` `init` to `__init__` central place